science.tex

```diff
@@ -161,29 +161,29 @@ \subsection{Probing Dark Energy and Dark Matter }
 The main LSST observables in the context of dark energy and matter will be:
 
 \begin{itemize}
-\item Measurements of the two-point auto- and cross-correlation of
-  shear across multiple redshift bins. %(Fig.~\ref{Fig:wlPk}).
-  There will be
-over 50 of these auto and cross power spectra constructed from the
-shape catalog with excellent photometric redshifts
+\item The {\sl 3x2-point} joint analysis galaxy--galaxy, galaxy--shear and shear--shear
+correlation functions that has become standard in precursor analyses (e.g DES Collaboration
+et al.~2017, Joudaki et al.~2017). WL and LSS are highly complementary probes, and the combination
+of their auto- and cross-correlations, which we spell out in the next three points, has a strong
+potential to constrain the properties of the late-time accelerated expansion.
+\item Measurements of the two-point auto- and cross-correlation of shear across multiple redshift bins.
+%(Fig.~\ref{Fig:wlPk}).
+There will be over 50 of these auto and cross power spectra constructed from the shape catalog with excellent photometric redshifts
 %correlation of shear of a galaxy at one redshift bin with another galaxy in
 %a different redshift bin, averaged over all pairs of billions of
 %galaxies;
 (Jain \& Taylor 2003).
-\item Higher-order shear and galaxy statistics that can improve dark energy
-constraints and provide self-calibration of various systematics (Takada \& Jain 2004;
-Dolney, Jain \& Takada 2006; Huterer et al.~2006). They are also probes of both
-primordial non-Gaussianities and those caused by non-linear structure.
-\item Similarly, weak lens magnification tomography (Morrison et al.~2012) offers a
-complementary probe of a mix of cosmic geometry and growth of dark matter structure.
 \item The galaxy-mass correlation function of billions of galaxies (as a function of their
```

Comment:
@damonge @slosar @rmandelb  : is it clear to you what is meant by galaxy-mass correlation function? (I know that this isn't your text). It could mean galaxy-galaxy clustering, or galaxy-galaxy lensing, both of which are mentioned separately below.

Reply:
Also, for consistency, both should be "power spectrum" or "correlation function." I'm happy to edit text, I just wasn't sure what was meant here.

Reply:
I'd be inclined to comment this old \item out entirely:  I think it is superceded by the new 3x2pt item from @damonge.

Reply:
I'll do that, and leave a note for @rmandelb to look at on Monday.

Reply:
@drphilmarshall : the idea was (after @mdschneider 's comments) that the first item I added would emphasize 3x2pt and the deep complementarity of LSS and WL, which was missing in the previous version. The next three points then spell out each of those 3 2-points (shear-shear, shear-galaxy and galaxy-galaxy). The item you've commented out would then correspond to the shear-galaxy part, so I would advocate for bringing it back (we could rephrase it a bit to make it more evident that it refers to shear-galaxy)

Reply:
Ah, good - I see. I uncommented and then made text edits in https://github.com/LSSTDESC/overview_paper/commit/c21da64ddb2a50577924c0dcaa1770a659f92a6a to do the rephrasing. Thanks!